src/components/EmptyContent/EmptyContent.vue

```diff
@@ -0,0 +1,76 @@
+<!--
+- @copyright Copyright (c) 2020 Greta Doci <gretadoci@gmail.com>
+-
+- @author 2020 Greta Doci <gretadoci@gmail.com>
+-
+- @license GNU AGPL version 3 or any later version
+-
+- This program is free software: you can redistribute it and/or modify
+- it under the terms of the GNU Affero General Public License as
+- published by the Free Software Foundation, either version 3 of the
+- License, or (at your option) any later version.
+-
+- This program is distributed in the hope that it will be useful,
+- but WITHOUT ANY WARRANTY; without even the implied warranty of
+- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+- GNU Affero General Public License for more details.
+-
+- You should have received a copy of the GNU Affero General Public License
+- along with this program. If not, see <http://www.gnu.org/licenses/>.
+-
+-->
+
+<template>
+	<div class="emptycontent icon-folder">
```

Comment:
The `icon` props should be used here to allow changing the icon, right?

Reply:
Yes, it shuld look more like something like that:
https://github.com/nextcloud/forms/blob/master/src/components/EmptyContent.vue

Reply:
done, please give it a general review so we can merge it